x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/dataframe/analyses/RegressionTests.java

```diff
@@ -131,7 +131,18 @@ public void testGetTrainingPercent() {
     public void testGetParams() {
         assertThat(
             new Regression("foo").getParams(null),
-            equalTo(Map.of("dependent_variable", "foo", "prediction_field_name", "foo_prediction")));
+            equalTo(Map.of("dependent_variable", "foo",
```

Comment:
nit: I think the code is more readable when all the map entries are left-aligned:
```suggestion
            equalTo(Map.of(
                "dependent_variable", "foo",
```